bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,32 @@
+#Bubble Sort in Python 3.x
+
+#Function to swap indices a and b in list arr
+
+def swap(arr, a, b):
+    x = arr[a]
+    arr[a] = arr[b]
+    arr[b] = x
+    
+# Function that performs bubble sort
+
+def bubble_sort(li):
+	for i in range(len(li)-1):
+		for j in range(len(li)-i-1):
+			if(li[j]>li[j+1]):
+				swap(li, j, j+1)
+
+
+	return li
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpbxs5zotg/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpbxs5zotg/bubble_sort/Python/bubble_sort.py
@@ -16,7 +16,7 @@
 				swap(li, j, j+1)
 
 
-	return li
+    return li
     
     
     
```